appveyor.yml

```diff
@@ -16,13 +16,26 @@ environment:
 install:
   - appveyor DownloadFile http://slproweb.com/download/Win32OpenSSL-1_0_1L.exe
   - Win32OpenSSL-1_0_1L.exe /silent /verysilent /sp- /suppressmsgboxes
+  - appveyor DownloadFile "http://sunet.dl.sourceforge.net/project/nsis/NSIS 2/2.46/nsis-2.46-setup.exe" -FileName nsissetup.exe
+  - nsissetup /S /D=%NSIS_ROOT%
 build:
 
 build_script:
   - md build
   - cd build
-  - cmake %CMAKE_ARGS% ..
-  - cmake --build .
+  - cmake -DCMAKE_BUILD_TYPE=Release %CMAKE_ARGS% ..
+  - cmake --build . --config Release
+
+after_build:
+  - cd ..
+  - cd installer
+  - '%NSIS_ROOT%\makensis -DVERSION=%APPVEYOR_BUILD_VERSION% libwebsockets.nsi'
```

Comment:
CMake actually has a builtin generator for CPack that generates NSIS stuff...

The benefit of using the built-in one is that the `INSTALL` targets in the CMake projects is used.
http://www.cmake.org/Wiki/CMake:CPackPackageGenerators#NSIS

This is not critical. But just so you know, and if you feel like changing it to use that instead.